website/source/docs/platform/k8s/helm.html.md

```diff
@@ -275,6 +275,8 @@ to run the sync program.
 
   * <a name="v-synccatalog-k8sprefix" href="#v-synccatalog-k8sprefix">`k8sPrefix`</a> (`string: ""`) - A prefix to prepend to all services registered in Kubernetes from Consul. This defaults to `""` where no prefix is prepended; Consul services are synced with the same name to Kubernetes. (Consul -> Kubernetes sync only)
 
+  * <a name="v-synccatalog-consulPrefix" href="#v-synccatalog-consulPrefix">`consulPrefix`</a> (`string: ""`) - A prefix to prepend to all services registered in Consul from Kubernetes. This defaults to `""` where no prefix is prepended. Consul services are synced with the same name to Kubernetes. (Kubernetes -> Consul sync only)
```

Comment:
```suggestion
  * <a name="v-synccatalog-consulPrefix" href="#v-synccatalog-consulPrefix">`consulPrefix`</a> (`string: ""`) - A prefix to prepend to all services registered in Consul from Kubernetes. This defaults to `""` where no prefix is prepended. Consul services synced one way to Kubernetes retain their names, which aren't customizable. 
```

Reply:
`Consul services synced one way to Kubernetes retain their names`
this should be from, right?